src/types/collection.graphql

```diff
@@ -481,11 +481,15 @@ input CollectionsInput {
 
 input CollectionInput {
   "The unique concept id assigned to the collection."
-  conceptId: String!
+  conceptId: String
```

Comment:
```suggestion
  conceptId: String!
```
any reason why we're removing this field as requried?

Reply:
They wanted to be able to find a collection with just the entryId and the provider. They didnt want to have to specify the conceptId.

Reply:
Who is 'they'?

@eudoroolivares2016 is right, every other singular (collection) query in the schema requires a conceptId. The plural version (collections) allows for more parameters like entryId and provider. We don't want to include this inconsistency to the project. Everything listed in the DAS-1958 ticket is already possible with the collections query.

This PR would also allow you to do a collection query with no parameters, which would fail. Or only providing a provider, which would return a single collection when more than 1 collection would be expected.

Reply:
They are the Data Access team, so David Auty and Owen Littlejohns. This is what DAS-1958 is about.  I will discuss this with them.